src/Services/Air/Air.js

```diff
@@ -409,16 +409,13 @@ module.exports = (settings) => {
           }
         ));
       };
-
+     
```

Comment:
Though our linter is not currently working for PRs, running it locally will not allow trailing spaces.

Please run linter locally and fix all errors/warnings

Reply:
I didn't fund any linter issue as I have run lint and test before raising pull request.

Reply:
Thank you for pointing this out, fixed. npm run lint and npm run test successfully passed